### PR TITLE
Create dockerimage.yml

### PR DESCRIPTION
**Pull request by olexii4**
Setup docker image build with github CI

### CODE REVIEW

**Review by akurinnoy — APPROVED** [review]

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: Docker Image CI
```
Comment (sleshchenko) [inline .github/workflows/dockerimage.yml]: Maybe it's better to name it as `Next Dockerimage`?
Next here would mean - the latest changes from the master branch

File: .github/workflows/dockerimage.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_FULL: quay.io/eclipse/che-dashboard-next:next
```
Comment (sleshchenko) [inline .github/workflows/dockerimage.yml]: It's not eclipse yet. 
```suggestion
      IMAGE_FULL: quay.io/che-incubator/che-dashboard-next:next
```

**Review by sleshchenko — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerimage.yml (hunk 1 above)
Comment (olexii4) [reply .github/workflows/dockerimage.yml]: Ok. I have fixed it.

File: .github/workflows/dockerimage.yml (hunk 2 above)
Comment (olexii4) [reply .github/workflows/dockerimage.yml]: Good catch. Fixed.

Comment (olexii4) [reply .github/workflows/dockerimage.yml]: Thanks

**Review by sleshchenko — APPROVED** [review]

**Pull request merged**